.github/workflows/update_aas_extension.yml

```diff
@@ -96,6 +123,17 @@ jobs:
             az resource create --resource-group $resourceGroupName --resource-type "Microsoft.Web/sites/siteextensions" --name "$aasProfilerOnlyName/siteextensions/DevelopmentVerification.DdDotNet.Apm" -p "{}"
             az resource create --resource-group $resourceGroupName --resource-type "Microsoft.Web/sites/siteextensions" --name "$aasProfilerBackendName/siteextensions/DevelopmentVerification.DdDotNet.Apm" -p "{}"
 
+            aasLinuxDotnetLatestName="dd-dotnet-linux-latest-build"
+            aasLinuxStatsName="dd-dotnet-linux-latest-build-stats"
+            aasLinuxProfilerDefaultName="dd-dotnet-linux-latest-build-profiler-default"
+            aasLinuxProfilerAllName="dd-dotnet-linux-latest-build-profiler-all"
```

Comment:
Before merging this PR, I'll create those apps: copying the environment variables from the windows ones.

Reply:
app created